### ISSUE TITLE
XP Multiplier

### ISSUE THREAD

**Issue by tylerecouture** [timberline-secondary/hackerspace#201]
Student suggestion: 

> Maybe if you had a consistent "streak" of correct quests without getting it returned, you could add a multiplier to XP. This may encourage kids to work a little bit harder and actually read the instructions better. Not only that, but it could be almost competitive as there could be a leader board for the students with a best streak. So for example, if I had a streak of 2, the multiplier would be something like 1.1. So that way, if I did a quest that gives 10XP, I would instead get 11.

**Comment by fjbagley** [comment]
I like the idea of rewarding reading the instructions more carfeully. The XP earned right now seems about right for the effort required. Perhaps there could be a leaderboard and a "loot box" system where students might get a random reward (pick the next VR game or customize your avatar). Further down the line, these rewards could even be traded.

**Comment by tylerecouture** [comment]
As I was looking at adding this in, I realized that this might be really annoying, in that instead of submitting their work when it's done, students will start requested teachers to double check everything before they submit it, to make sure they keep up their streak.  Would that be too annoying?

**Comment by fjbagley** [comment]
Yes, I think it could be annoying. It takes a few weeks to get some of the double checkers to stop as is. Sometimes students don't quite get an answer right but were close and losing the multiplier could be frustrating. Also, the amount of XP provided is very fair. 

Students that get the answers right the first time are able to continue in the quest line quicker a reward in itself. I already have a bit of a bi-modal distribution. If I had the option of an XP multiplier at Studio Tyee, I would disable it.
